lib/build-tokens.js

```diff
@@ -56,9 +57,14 @@ async function buildTokensCommand(commandArgs) {
     source: tokensSource
       ? [`${tokensSource}/core/**/*.json`, `${tokensSource}/core/**/*.toml`]
       : [],
+    preprocessors: ['tokens-studio'],
+    expand: {
+      typesMap: (await getTokensStudioTransforms()).expandTypesMap,
```

Comment:
[inform] See `sd-transforms` README for more information on [`expandTypesMap`](https://github.com/tokens-studio/sd-transforms?tab=readme-ov-file#using-expand) for `expand`.

Reply:
Related to https://github.com/openedx/paragon/pull/3222